x-pack/plugins/security/server/routes/api/v1/__tests__/authenticate.js

```diff
@@ -295,17 +295,15 @@ describe('Authentication routes', () => {
       const unhandledException = new Error('Something went wrong.');
       serverStub.plugins.security.authenticate.throws(unhandledException);
 
```

Comment:
__note:__ actually these tests were potentially buggy, we should never have assertions in a `catch` **or** `then`, if we have main assertions in `catch` then `then` should fail the test if it's called (noticed that because one of the tests was green, but I expected it to be red :)).